swagger/v3/component_schemas/ParticipantDeclarationAttributes.yml

```diff
@@ -105,6 +105,14 @@ properties:
       - training-event-attended
       - self-study-material-completed
       - other
+      - "<br>For cohort 2025, depending on the declaration type the new possible values are:"
```

Comment:
not sure about this, so this will appear correctly in the api reference however will break the [api spec](https://cpd-ecf-review-5599-web.test.teacherservices.cloud/lead-providers/api-docs/v3/api_spec.yml). I think we'll need to amend, and maybe explain in the description the difference however have on enum, let's dicsus

Reply:
discussed in person: we'll need to amend the content itself, so instead we'll have the full enum and we'll just move the following content to description:
```
For cohort 2025, depending on the declaration type there are different possible values.
```
Maybe it's worth checking with Mark incase on this?